mock/editors/tools.md

```diff
@@ -0,0 +1,26 @@
+# Document Creation and Tools
+## Menu
+* [Style for Group-internal Drafts](#) (below)
+* [ReSpec and BikeShed](#) (below)
+* [Pubrules](#) (below)
+* [Normative References](#) (below)
+* [<- Back to **Editors**](index.md#)
+
+## Content
+Here find all the information you need on being an Editor for W3C Rec Track Documents and Notes.
+
+@todo: check all tools carried from here: https://www.w3.org/2003/Editors/
+
+### Style for Group-internal Drafts
+Take and modify content from <https://www.w3.org/2005/03/28-editor-style.html> and <https://w3c.github.io/manual-of-style/>
+
+### ReSpec and BikeShed
+Take and modify content from <https://tabatkins.github.io/bikeshed/>, <https://github.com/w3c/respec/wiki>
+
+### Pubrules
+(Publication Requirements) and links to related policies (e.g., namespaces, MIME type registration, version management, and in-place modifications)
```

Comment:
Add pubrules checker and other validation tools e.g. html validation, link checker etc

Reply:
Added a note in the document to make sure these are added in the pubrules section